docs/articles_en/openvino-workflow/model-optimization-guide/weight-compression.rst

```diff
@@ -161,15 +161,15 @@ trade-offs after optimization:
   `Larger Group Size`: Results in faster inference and a smaller model, but might
   compromise accuracy.
 
-* ``ratio`` controls the ratio between INT4 and INT8_ASYM compressed layers in the model.
+* ``ratio`` controls the ratio between INT4 and layers compressed to ``backup_mode`` in the model.
```

Comment:
Let's fix `INT4` as well as it can be NF4 and others in the future. You can rephase it to:
```suggestion
* ``ratio`` controls the ratio between the layers compressed to the precision defined by ``mode`` and the rest of the layers that will be kept in the ``backup_mode`` in the optimized model.
```